0001_education_matters_addChild.js

```diff
@@ -0,0 +1,72 @@
+const Common = require("../../../../setup/common.js");
+
+const folderName = __dirname.match(/[^\\/]+$/);
+Cypress.on("uncaught:exception", () => false);
+Cypress.on("uncaught:exception", (err) => {
```

Comment:
`Cypress.on("uncaught:exception", () => false);`
This will cover all errors so no need fort the second error handler.

Reply:
It's solved